Logel's database

lizofall opened the pull request.

lizofall committed:
Logel's database

JeanPhilippeFall commented:
[codification.sql.txt](https://github.com/CheikhMbacke/Logel/files/4365635/codification.sql.txt)
Fichier de la base de données de Logel
Création de 4 rôles : admin, chefDePav, Etudiant, Comptable, chargéDeLingerie
Attribution des privilèges pour chaque rôle
Création de 4 types d'utilisateurs distants de la base de données: admin, CDP, etu, compt, CDL auquels on rattache les rôles

JeanPhilippeFall committed:
Database file

Dans le fichier, vous verrez les différents rôles : admin, chefDePav, Etudiant, chargéDeLingerie, Comptable
Chaque rôle détient son lot de privilège et est associé à un type d'utilisateur distant. Voir dans la dernière partie du fichier

lizofall commented:
t'as déja fini avec ça? @JeanPhilippeFall

JeanPhilippeFall committed:
Add files via upload

JeanPhilippeFall commented:
Ouai! Ya need help ?
